src/main/java/org/jabref/gui/util/WelcomePage.java

```diff
@@ -0,0 +1,81 @@
+package org.jabref.gui.util;
+
+import javafx.geometry.Pos;
+import javafx.scene.control.Hyperlink;
+import javafx.scene.control.Label;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Color;
+import javafx.scene.text.Font;
+import javafx.scene.text.Text;
+import javafx.scene.text.TextAlignment;
+import javafx.scene.text.TextFlow;
+
+import org.jabref.gui.ClipBoardManager;
+import org.jabref.gui.DialogService;
+import org.jabref.gui.StateManager;
+import org.jabref.gui.frame.JabRefFrame;
+import org.jabref.gui.importer.NewDatabaseAction;
+import org.jabref.gui.importer.actions.OpenDatabaseAction;
+import org.jabref.gui.preferences.GuiPreferences;
+import org.jabref.gui.undo.CountingUndoManager;
+import org.jabref.logic.ai.AiService;
+import org.jabref.logic.util.TaskExecutor;
+import org.jabref.model.entry.BibEntryTypesManager;
+import org.jabref.model.util.FileUpdateMonitor;
+
+/**
+ * A simple Welcome Page for the JabRef application.
+ */
+public class WelcomePage extends VBox {
+
+    public WelcomePage(JabRefFrame frame,
+                       GuiPreferences preferences,
+                       AiService aiService,
+                       DialogService dialogService,
+                       StateManager stateManager,
+                       FileUpdateMonitor fileUpdateMonitor,
+                       BibEntryTypesManager entryTypesManager,
+                       CountingUndoManager undoManager,
+                       ClipBoardManager clipBoardManager,
+                       TaskExecutor taskExecutor) {
+
+        setAlignment(Pos.CENTER);
+        setSpacing(10);
+
+        // Title
+        Label welcomeLabel = new Label("Welcome to JabRef!");
+        welcomeLabel.setFont(new Font("Arial", 40));
```

Comment:
Can you please add all the font and color stuff to the Base.css file?
It should also look nice with dark theme and customizable by the user

Reply:
Thank you! I have moved the styling to `Base.css`. See screenshots below for light and dark modes:

![image](https://github.com/user-attachments/assets/f8983f28-050b-45d0-abb3-bd61ddff51a0)

![image](https://github.com/user-attachments/assets/cf4e0c41-3237-4eaf-bdd5-1afb32cc1971)

Let me know if anything further needs to be changed.


https://github.com/JabRef/jabref/pull/12017/commits/5d50f08eee4465087f6e8cfb8407fef90387491a